### DISCUSSION TITLE
Yearly Subscription to GitHub Copilot Suddenly Not Accessible

### DISCUSSION THREAD

**Discussion by gtoxlili** [community/community#75338] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello GitHub Community,

I've encountered a problem with my GitHub Copilot subscription and I'm hoping to get some insights or assistance here. I have an active annual subscription, but out of the blue, I've been barred from using the service. This has been quite unexpected and is adversely affecting my productivity.

I've been relying on Copilot for my daily coding tasks and have not made any changes to my account or payment method that could have resulted in this issue. There wasn't any prior communication about potential issues or reasons for termination. 

Has anyone else faced this kind of issue? What steps can I take to resolve this? Any advice or direction would be greatly appreciated as I'm currently at a standstill without access to Copilot.

Thank you in advance for your help.

**Comment by b8zeek** [comment]
I've seen this happening before, the main reason was the payment processing issues for the users at the beginning of their subscriptions. I am not sure if that's a case with you but the discussions is not the right place for finding the resolution; make sure to reach out to GitHub Support team, link below.

https://support.github.com/

Good luck with this. 🤞🏼 

🍀

**Reply by gtoxlili** [reply]
Thanks for getting back to me. 

However, my issue is that I bought an annual subscription to GitHub Copilot about two months ago, and suddenly I can't use it these days. It says, "Your GitHub Copilot subscription has ended." I have submitted a ticket but haven't received a response, so I want to see if anyone else has experienced the same situation. 

I lean pretty heavily on Copilot for my day-to-day coding, so this hitch is throwing a wrench in my workflow big time, and it's got me on edge :(

**Reply by b8zeek** [reply]
I am sorry this is happening to you.

BUT!!! This is also an amazing lesson for you, you are leaning too much on the AI if you're not able to perform w/o it! It should be there to ease some tasks, speed you up a bit for you not to substitute you.

You should re-consider your approach and how you use the tool. At the end, it is a tool, it is not you!

Best of luck bro, hope the Support will get to you ASAP, they usually are quick! 🍺 

🍀

**Comment by gtoxlili** [comment, marked as answer]
They refunded me the money
